tests/test_matches.py

```diff
@@ -389,8 +389,10 @@ def test_state(self):
         self.chain.insert_rule(self.rule)
         rule = self.chain.rules[0]
         m = rule.matches[0]
-        self.assertEquals(m.name, "state")
-        self.assertEquals(m.state, "RELATED,ESTABLISHED")
+        self.assertEqual(m.name, "state")
+        self.assertEqual(m.state, "RELATED,ESTABLISHED")
+        self.assertEqual(rule.matches[0].name, self.rule.matches[0].name)
+        self.assertEqual(rule, self.rule)
```

Comment:
@ldx , this is the test that would fail with older versions since it compares head to head the 2 rules.

Reply:
Line 395: self.assertEqual(rule, self.rule)